docs/src/content/docs/id/guides/sidebar.mdx

```diff
@@ -51,31 +49,85 @@ Pelajari lebih lanjut tentang sidebar yang dihasilkan secara otomatis di bagian
 
 ## Tambahkan tautan dan grup tautan
 
-Untuk mengonfigurasi [tautan](#tautan) dan [grup tautan](#grup-tautan) (dalam header yang dapat dilipat), gunakan properti [`starlight.sidebar`](/id/reference/configuration/#sidebar) dalam `astro.config.mjs`.
+Untuk mengonfigurasi tautan sidebar dan grup tautan (dalam header yang dapat dilipat), gunakan properti [`starlight.sidebar`](/id/reference/configuration/#sidebar) di `astro.config.mjs`.
 
 Dengan menggabungkan tautan dan grup, Anda dapat membuat berbagai macam layout sidebar.
 
-### Tautan
+### Tautan Internal
 
-Tambahkan tautan ke halaman internal atau eksternal menggunakan objek dengan properti `label` dan `link`.
+Tambahkan tautan ke halaman di `src/content/docs/` menggunakan objek dengan properti `slug`.
+Judul halaman yang ditautkan akan digunakan sebagai label secara default.
 
-```js
+Misalnya, dengan konfigurasi berikut:
+
+```js "slug:"
 starlight({
 	sidebar: [
-		// Tautan ke panduan CSS dan Styling.
-		{ label: 'CSS dan Styling', link: '/guides/css-and-tailwind/' },
-		// Tautan eksternal ke website Astro.
-		{ label: 'Astro', link: 'https://astro.build/' },
+		{ slug: 'constellations/andromeda' },
+		{ slug: 'constellations/orion' },
 	],
 });
 ```
 
-Konfigurasi di atas akan menghasilkan sidebar berikut:
+Dan struktur berkasnya sebagai berikut:
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - constellations/
+        - andromeda.md
+        - orion.md
+
+</FileTree>
+
+Sidebar berikut akan dibuat:
 
 <SidebarPreview
 	config={[
-		{ label: 'CSS dan Styling', link: '/id/guides/css-and-tailwind/' },
-		{ label: 'Astro', link: 'https://astro.build/' },
+		{ label: 'Andromeda', link: '' },
+		{ label: 'Orion', link: '' },
+	]}
+/>
+
+Untuk mengganti nilai yang disimpulkan dari frontmatter halaman yang ditautkan, Anda dapat menambahkan properti `label`, [`translations`](#internasionalisasi), dan [`attrs`](#atribut-html-custom).
+
+Lihat [“Menyesuaikan tautan yang dibuat secara otomatis”](#menyesuaikan-tautan-yang-dihasilkan-secara-otomatis-di-frontmatter) untuk detail lebih lanjut tentang cara mengendalikan tampilan sidebar dari halaman frontmatter.
+
+#### Singkatan untuk tautan internal
+
+Tautan internal juga dapat ditentukan dengan hanya menyediakan string untuk slug halaman sebagai singkatan.
+
+Misalnya, konfigurasi berikut ini setara dengan konfigurasi di atas, yang menggunakan `slug`:
+
+```js "slug:"
+starlight({
+	sidebar: ['constellations/andromeda', 'constellations/orion'],
+});
+```
+
+### Tautan lainnya
+
+Tambahkan tautan ke halaman eksternal atau non-dokumen menggunakan objek dengan properti `label` dan `link`.
+
+```js "label:" "link:"
+starlight({
+	sidebar: [
+		// A link to a non-docs page on this site.
```

Comment:
This comment can be translated.

Reply:
Ah i miss this one, thank you for the review. Will update soon.